protocol/tx/spend.go

```diff
@@ -9,17 +9,43 @@ type spend struct {
 		ExtHash     bc.Hash
 	}
 	ordinal int
+
+	// SpentOutput contains (a pointer to) the manifested entry
+	// corresponding to body.SpentOutput. It may be nil, in which case
+	// spentInfo must be non-nil.
+	SpentOutput *output
+
+	// spentInfo contains validation-essential data in case the spent
+	// output object is not present (in SpentOutput).
```

Comment:
No, that's not right. In an all-txgraph world, we'll still be constructing spends from utxos in the database, and those don't contain refdata (nor should they - unless you really hate the `spentInfo` type, and [like me] you also hate the idea of an incompletely populated `output` object in the `spend`).